soundchan.properties.example

```diff
@@ -6,3 +6,9 @@ localFilePath=C:\\PATH\\TO\\SOUNDS\\DIRECTORY
 
 //The user for the 
 followingUser=USERNAME
+
+//If you want SoundChan to play an audio file whit their name when a user joins the channel or have that information come from below file
+audioOnUserJoin=on/off
```

Comment:
Maybe have this be a true false boolean instead?

Reply:
I have a function that checks for enable/on/true/1 to make the property true, since I didn't know what we wanted to use